.github/workflows/bazel.yml

```diff
@@ -0,0 +1,92 @@
+name: Build
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  macos_build:
+
+    runs-on: macos-11.0
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Write local.bazelrc File
+      shell: bash
+      run: |
+        cat >local.bazelrc <<EOF
```

Comment:
isn't this where ci.bazelrc would be used?

Reply:
It is. The `common --config=ci` tells Bazel to use the `ci` config values.